overlord/snapstate/snapmgr.go

```diff
@@ -139,6 +139,11 @@ type SnapState struct {
 	// InstanceKey is set by the user during installation and differs for
 	// each instance of given snap
 	InstanceKey string `json:"instance-key,omitempty"`
+
+	// RefreshPostponedTime records the time when the refresh was first
+	// attempted but postponed because the snap was busy. This value is
+	// reset on each successful refresh.
+	RefreshPostponedTime time.Time `json:"refresh-postponed-time,omitempty"`
```

Comment:
my main wondering here is about postponed, it sounds a bit too benign and also it's mostly clear but not completely whether this is the time it was postponed at vs to.   We could express the blockage instead  with "blocked", "inhibited", or "held".

Reply:
I like "inhibited" fwiw

Reply:
That works for me  over "postponed"

Reply:
Renamed.